build.gradle

```diff
@@ -36,6 +36,12 @@ jar {
     }
 }
 
+run {
+    if (project.hasProperty("appArgs")) {
```

Comment:
Please add a comment here, how this can be used to pass arguments to signal-cli.

Reply:
okay